.github/workflows/ghcr-image-build-and-publish.yml

```diff
@@ -7,11 +7,11 @@ name: Container Image Build
 
 on:
   push:
-    branches: [ ]
+    branches: [ master ]
     # Publish semver tags as releases.
     tags: [ 'v*.*.*' ]
-  pull_request:
-    branches: [ ]
+#   pull_request:
+#     branches: [ master ]
```

Comment:
Seems accidentally commented out?

Reply:
No knowingly commented to doesn't built image on PR

Reply:
Can we just remove these lines?

Reply:
Done as requested @AkihiroSuda